labman/db/process.py

```diff
@@ -2853,7 +2853,7 @@ def _sequencer_i5_index(sequencer, indices):
     @staticmethod
     def _format_sample_sheet_data(sample_ids, i7_name, i7_seq, i5_name, i5_seq,
                                   sample_projs, wells=None, sample_plates=None,
-                                  description=None, lanes=[1], sep=',',
+                                  description=None, lanes=None, sep=',',
```

Comment:
That is a great fine, but also concerning given the type of bugs that could be introduced in a long running system. Do you think an issue should be created to determine whether other defaults like this were set?